docs/source/software/quick_start_guide.rst

```diff
@@ -108,5 +108,33 @@ QGroundControl and change the following:
 * EKF2_HGT_MODE = vision
 
 
-Safety  
-=======================
\ No newline at end of file
+Safety Checklist
+==================
+
+Make sure you complete the following check list *before every flight*. Most of the checks are automated by the controller on startup, but that is no substitute for good ole human caution.
+
+
+Hardware
+----------
+
+1. Mechanical
+    a. All *4* rotors are tightly secured
+    b. Battery is secured 
```

Comment:
Do we need to be specific about what "secured" means? Since for the rotors, it seems that twisting the bolts by hand isn't always enough for longer term use.

Reply:
done

Reply:
This is much more clear now!